.flake8

```diff
@@ -102,6 +102,10 @@ exclude =
     venv
     *.md
     src/webapp/node_modules/
+    src/meltano/api/events/notification_events.py
+    src/meltano/core/logging/job_logging_service.py
+    src/meltano/core/job/stale_job_failer.py
+    tests/meltano/core/runner/test_runner.py
```

Comment:
These had not been touched in a while so flake8 is complaining. Happy to un-ignore them.

Reply:
That's okay. I think it's fine to leave lint cleanup for another day.

Thanks. 👍